TicketBookingAPI/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "Connect": "Data Source=DD-THINHHN;Initial Catalog =TicketBooking; Integrated Security =True"
```

Comment:
This connection string only works on `DD-THINHHN` machine. it should work for everyone.

Reply:
suggest change `Connect` to `DefaultConnection`